### PR TITLE
Revert "Font-family declarations fixed"

### PR DESCRIPTION
**Pull request by lauriejim**
Reverts strapi/strapi#1928

Needed by the design team.

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/LeftMenuFooter/styles.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
   padding-left: 15px;
   padding-right: 15px;
   line-height: 3rem;
-  font-family: 'Lato', sans-serif;
+  font-family: 'Lato';
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/components/LeftMenuFooter/styles.scss]: Remove quote?

File: packages/strapi-admin/admin/src/styles/base/fonts.scss [hunk 2]
```diff
@@ -157,79 +157,79 @@
 
 /* OpenSans (light, regular) */
 @font-face {
-  font-family: 'OpenSans', sans-serif;
+  font-family: 'OpenSans';
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/styles/base/fonts.scss]: Why there are quotes here and not for Lato?

File: packages/strapi-plugin-content-manager/admin/src/components/Wysiwyg/componentsStyles.scss [hunk 3]
```diff
@@ -6,14 +6,14 @@
...
-  font-family: 'OpenSans', sans-serif;
+  font-family: "OpenSans";
   cursor: text;
 
   // TODO define rules for header's margin
   h1, h2, h3, h4, h5, h6 {
     margin: 0;
     line-height: 24px !important;
-    font-family: 'OpenSans', sans-serif;
+    font-family: "OpenSans";
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/components/Wysiwyg/componentsStyles.scss]: Why there are double quotes here and simple ones above?

**Comment by lauriejim** [comment]
@Aurelsicoko This PR is just a revert. Answer to all your comments is "It was this state before this PR"

But we can make updates to clean it.

**Comment by Aurelsicoko** [comment]
You can merge it though... We'll provide a theme system in the future with a new components library to easily custom colours, fonts, etc. We'll harmonise everything in the future 👍

**Pull request merged**